spec/cucumber/cli/rerun_spec.rb

```diff
@@ -71,16 +71,17 @@ module Cli
         end
       end
 
-      context "contains features and scenarios with and without spaces same line and across lines" do
+      context "contains features and scenarios with and without spaces on same line and across lines" do
```

Comment:
@mattwynne Parallel_test's Cucumber logger now creates this output without any space. It used to work well but this commit broken it https://github.com/grosser/parallel_tests/pull/416  
related issue can be found here https://github.com/grosser/parallel_tests/pull/398